units/src/amount.rs

```diff
@@ -109,13 +122,12 @@ impl FromStr for Denomination {
 
     /// Converts from a `str` to a `Denomination`.
     ///
-    /// # Accepted Denominations
+    /// # Errors
     ///
-    /// All upper or lower case, excluding SI prefix (c, m, u) which must be lower case.
-    /// - Singular: BTC, cBTC, mBTC, uBTC
-    /// - Plural or singular: sat, satoshi, bit
+    /// - If the denomination begins with a capital `M` a `PossiblyConfusingDenominationError` is
+    ///   returned.
     ///
-    /// Due to ambiguity between mega and milli we prohibit usage of leading capital 'M'.
+    /// - If an unknown denomination is used, an `UnknownDenominationError` is returned.
```

Comment:
This whole doc should be moved on top of `impl` block to be more visible. Nobody will see it on a trait method.

But I don't really want to review this again, so I'd prefer it done in a followup PR.

Reply:
Sorry I had to remove the line wrap change and lost your approval.  I have not made any other changes and will look into the suggested doc change seperately.